scale/job/test/execution/configuration/test_configurators.py

```diff
@@ -318,7 +332,26 @@ def test_configure_queued_job_ingest_with_new_workspace(self, mock_msg_mgr):
         from ingest.test import utils as ingest_test_utils
         scan = ingest_test_utils.create_scan()
         ingest = ingest_test_utils.create_ingest(scan=scan, workspace=workspace_1, new_workspace=workspace_2)
-        Ingest.objects.start_ingest_tasks([ingest], scan_id=scan.id)
+
+        # start ingest_tasks (can't call method due to using command messages)
```

Comment:
make a message object and call it's execute method to save on code duplication?

Reply:
I didn't even think of that. Good idea.